bin/cocitations.py

```diff
@@ -1,9 +1,22 @@
 import multiprocessing
 from arxiv_public_data import internal_citations
+from arxiv_public_data.config import DIR_FULLTEXT
 
 if __name__ == '__main__':
     import sys
-    processes = multiprocessing.cpu_count() if len(sys.argv) <= 1 else int(sys.argv[1])
 
-    cites = internal_citations.citation_list_parallel(N=processes)
+    opts = [opt for opt in sys.argv[1:] if opt.startswith("-")]
+    args = [arg for arg in sys.argv[1:] if not arg.startswith("-")]
+    opts_args = dict(zip(opts, args))
+    if '-dir' in opts_args:  # option for directory of full text files
+        txt_dir = opts_args['-dir']
```

Comment:
The CLI flag style is typically `--dir` for flags/parameters denoted by multiple characters.

Reply:
If you want to parse these args you may as well use the `argparse` library so there is some documentation built in for the user.

Reply:
something like:
```python
from argparse import ArgumentParser

parser = ArgumentParser(description="""describe the tool""")
parser.add_argument("N", type=int, default=multiprocessing.cpu_count(), help="describe this parameters")
parser.add_argument("--dir", type=str, default=DIR_FULLTEXT, help="describe this parameter flag")

args = parser.parse_args()

cites = internal_citations.citation_list_parallel(N=args.N, directory=args.dir)
```